static/styles/message_row.css

```diff
@@ -80,9 +80,15 @@ $sender_name_distance_below_flex_center: 3px;
         .message_content {
             grid-row-start: 1;
             grid-column-start: 2;
-            padding: 4px 0 1px;
+            padding: 0;
+            padding-top: 2px;
+
+            p {
+                margin: 1px 0 2px 0;
```

Comment:
This overrides `margin: 3px 0`, so paragraphs in messages will now be closer together. 

If we don't want that, I can try to find another way to remove space from inside the focus-border (which we need to do to keep the overall message height the same) but this was the best thing I could find to do so far.

Reply:
Yeah, this is a red flag, we don't want to change anything inside `message_content` without a thorough process that it doesn't break 10 other things.

Reply:
Ah yeah okay fair. Do you have any ideas for places we can pull space from to keep the height the same?

Also do you think the padding changes here are okay or would you be worried about that too?

Reply:
I can take a look, can you rebase it?

Reply:
done!

Reply:
I think you can make a case here to increase the overall height of the message or discuss it with Vlad in #desing with a proper screenshot of how elements are occupying the space.

Reply:
done, thanks! https://chat.zulip.org/#narrow/stream/101-design/topic/redesign.3A.20.20focus.20rectangle/near/1491642